config-model/src/main/resources/schema/deployment.rnc

```diff
@@ -7,13 +7,37 @@ start = element deployment {
    attribute major-version { text }? &
    attribute athenz-domain { xsd:string }? &
    attribute athenz-service { xsd:string }? &
-   Upgrade? &
-   BlockChange* &
-   Notifications? &
-   Endpoints? &
-   Test? &
-   Staging? &
-   Prod*
+   Step
+}
+
+Step =
+    StepExceptInstance &
+    Instance*
+
+StepExceptInstance =
+    Delay* &
+    ParallelInstances* &
+    Upgrade? &
+    BlockChange* &
+    Notifications* &
+    Endpoints? &
+    Test? &
+    Staging? &
+    Prod*
+
+Instance = element instance {
+   attribute id { xsd:string } &
+   attribute athenz-domain { xsd:string }? &
+   attribute athenz-service { xsd:string }? &
+   StepExceptInstance
```

Comment:
Can't really have nested instances, but expressing that (this StepExceptInstance can't have ParallelInstances) seems like overkill.

Reply:
Yes, agreed.